paper-input-container.html

```diff
@@ -116,30 +116,35 @@
 `display:inline-block`.
 -->
 
+<custom-style>
+  <style is="custom-style">
+    html {
```

Comment:
In case we want to backport it to 1.x, we might need to add `:root` here

Reply:
i'll add it just in case

Reply:
oh tldr not needed (this is a 1.x PR, let's cross that bridge when we get there)